src/index.js

```diff
@@ -1 +1,43 @@
-'use strict';
+// 'use strict';
+
+// import express from 'express';
```

Comment:
The import statement for Express is commented out. Uncomment this line to use Express in your application.